packages/postgres-database/src/simcore_postgres_database/models/projects_tags.py

```diff
@@ -13,8 +13,13 @@
     sa.Column(
         "project_id",
         sa.BigInteger,
-        sa.ForeignKey(projects.c.id, onupdate="CASCADE", ondelete="CASCADE"),
-        nullable=False,
+        sa.ForeignKey(
+            projects.c.id,
```

Comment:
Q: do you know why we use the project ID instead of the UUID here?

Reply:
looks like it should be the uuid here.

Reply:
I am just changing nullable from False to True. I have not modified the project field. As for why `project_id` was used instead of `project_uuid`, I do not know (it wasn’t introduced by me). However, it is a valid ID field, so it is fine.